docs/02-app/02-api-reference/02-file-conventions/route.mdx

```diff
@@ -53,12 +53,26 @@ The `request` object is a [NextRequest](/docs/app/api-reference/functions/next-r
 
 ### `context` (optional)
 
-```ts filename="app/dashboard/[team]/route.js"
+```ts filename="app/dashboard/[team]/route.ts" switcher
+type Params = {
+  team: string;
+}
+
+export async function GET(request: Request, context: { params: Params }) {
+  const team = context.params.team // '1'
```

Comment:
```suggestion
  const team = params.team // '1'
```
It should be this due to object destructuring.

Reply:
Not sure if i'm missing something here - but I don't think the you get params destructured here? I had to call context.params.team when i used it or alternatively `const { team } = context.params;`

Reply:
@chironjit Yes my apologies!